EZP-31389 Document built-in URL criteria in URLService (#1061)
ezsystems/developer-documentation: docs/api/url_reference/logicalnot_criterion.md

# LogicalNot Criterion

The [`LogicalNot` URL Criterion](https://github.com/ezsystems/ezplatform-kernel/blob/v1.0.0/eZ/Publish/API/Repository/Values/URL/Query/Criterion/LogicalNot.php)

> matches a URL if the provided criterion does not match.

Reviewer: ```suggestion
matches a URL if the provided Criterion does not match.
```

> It takes only one criterion in the array parameter.

Reviewer: ```suggestion
It takes only one Criterion in the array parameter.
```

## Arguments

> - `criterion` - represents the criterion that should be excluded from the results

Reviewer: ```suggestion
- `criterion` - represents the Criterion that should exclude a URL from the results
```
not sure about this, but maybe clearer this way

Author: ```suggestion
- `criterion` - represents the Criterion that should be negated
```

## Example

``` php

> $query->query = new new Criterion\LogicalNot(
>         new Criterion\Pattern('ez.no')
> );

Reviewer 2: ```suggestion
$query->filter = new new Criterion\LogicalNot(
        new Criterion\Pattern('ez.no')
);
```
